src/components/Header/Header.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
```

Comment:
As of React 17 and later, it is generally no longer necessary to explicitly import when using JSX `React`. If there are TypeScript errors due the ChangeEvent and KeyboardEvent usage, try to use `import { useState, ChangeEvent, KeyboardEvent } from 'react';` instead with corresponding changes for event handlers.